curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67332fff82eb28128f08ec2c.md

```diff
@@ -0,0 +1,87 @@
+---
+id: 67332fff82eb28128f08ec2c
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+To the `.card:hover` selector add a `transform` property set to `scale(1.1)`.
```

Comment:
It looks like `transform` was briefly mentioned at the end of the pseudo elements script.
But it wouldn't hurt to have an explanation with a code snippet here to as a reminder since it only came up once and they might have forgotten.